dbus_next/aio/proxy_object.py

```diff
@@ -141,9 +144,7 @@ async def property_setter(val):
 
             BaseProxyInterface._check_method_return(msg)
 
-        snake_case = BaseProxyInterface._to_snake_case(intr_property.name)
-        setattr(self, f'get_{snake_case}', property_getter)
-        setattr(self, f'set_{snake_case}', property_setter)
```

Comment:
One more minor thing(not the primary motivation however) this change helps with is making IDEs not get confused by the use of setattr(which seems to make them think attributes that exist don't exist since they aren't loaded yet) when it comes to determining if a method on the object exists.